tests/i18n/test_es.py

```diff
@@ -94,18 +94,9 @@ def test_returns_true_on_valid_nif(value: str):
     assert es_nif(value)
 
 
-@pytest.mark.parametrize(
-    ("value",),
-    [
-        ("12345",),
-        ("X0000000T",),
-        ("00000000T",),
-        ("00000001R",),
```

Comment:
If these are valid, can you please put them in the valid `nif` test cases?

Reply:
Done